pkg/dao/repository_configs.go

```diff
@@ -199,6 +201,12 @@ func (r repositoryConfigDaoImpl) List(
 		filteredDB = filteredDB.Where(orGroup)
 	}
 
+	if filterData.Status != "" {
+		statusesString := cases.Title(language.Und).String(filterData.Status)
+		statuses := strings.Split(statusesString, ",")
```

Comment:
I made the filtering not case sensitive. I'm not sure if we want that, but it's easy to remove.

Reply:
Sounds like we likely don't unfortunately (as discussed during standup).